spring/spring4/spring4-embedded/src/test/java/org/infinispan/spring/provider/SpringCacheTest.java

```diff
@@ -266,6 +266,28 @@ public void testValueLoaderWithExceptionWhenLoading() {
       cache.get("test", () -> {throw new IllegalStateException();});
    }
 
+   @Test
+   public void testPutNullAndGetWithClass() {
+      //when//then
```

Comment:
Is this really useful?

Reply:
Both of three tests were different errors

Reply:
I meant the comment, IMO it doesn't add anything